docs/src/docs/asciidoc/gradle-plugin.adoc

```diff
@@ -356,8 +356,8 @@ See <<configuration-options>> for the full list of available options.
 [[metadata-support]]
 == GraalVM Reachability Metadata Support
 
-Since release 0.9.11, the plugin adds experimental support for the https://github.com/oracle/graalvm-reachability-metadata/[GraalVM reachability metadata repository].
-This repository provides https://www.graalvm.org/22.2/reference-manual/native-image/ReachabilityMetadata/[reachability metadata] for libraries that do not support GraalVM Native Image.
+The plugin adds experimental support for the https://github.com/oracle/graalvm-reachability-metadata/[GraalVM reachability metadata repository].
```

Comment:
It the support for reachability metadata  still experimental?

Reply:
I don't think so. cc @melix

Reply:
No, it's not experimental anymore.